index.html

```diff
@@ -370,7 +371,7 @@
             </div>
         </div>
 
-        <p id="prestigeshardinfo" alt="prestigeshardinfo" label="prestigeshardinfo" style="color:lightcyan; text-align: center">You have 1.39e+100394 Prestige Shards, providing 1,214,593 multiplier power boosts.</p>
+        <p id="prestigeshardinfo" alt="prestigeshardinfo" label="prestigeshardinfo" style="color: #a7ffff; text-align: center">You have 1.39e+100394 Prestige Shards, providing 1,214,593 multiplier power boosts.</p>
```

Comment:
should this color still be hardcoded?

Reply:
You mean whats wrong with lightcyan? Its to close to white, #a7ffff is slighly darker